<launch>
    <arg name="dataset_folder"  default="/KITTI/" />
    <arg name="sequence_number" default="00" />
    <arg name="to_bag"          default="true" />
    <arg name="output_bag_file" default="$(arg dataset_folder)bag/kitti$(arg sequence_number).bag" />
    <arg name="publish_delay"   default="1" />

    <node name="kittiHelperWithGt" pkg="aloam_velodyne" type="kittiHelperWithGt" output="screen"> 
        <param name="dataset_folder"    type="string"   value="$(arg dataset_folder)" />
        <param name="sequence_number"   type="string"   value="$(arg sequence_number)" />
        <param name="to_bag"            type="bool"     value="$(arg to_bag)" />
        <param name="output_bag_file"   type="string"   value="$(arg output_bag_file)" /> <!-- replace with your output folder -->
        <param name="publish_delay"     type="int"      value="$(arg publish_delay)" />
    </node>
</launch>